SmallStatusWidget/LoopCircleView.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  LoopCircleView.swift
+//  Loop
+//
+//  Created by Noah Brauner on 8/15/22.
+//  Copyright © 2022 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+import LoopCore
+
+struct LoopCircleView: View {
+    var entry: SmallStatusEntry
+
+    var body: some View {
+        let closeLoop = entry.closeLoop
+        let lastLoopCompleted = entry.lastLoopCompleted ?? Date()
```

Comment:
Should not assume fresh loop, if nothing was stored in UserDefaults for lastLoopCompleted.

Reply:
Done, it now shows a red loop if nothing was stored in UserDefaults for lastLoopCompleted.